rllib/policy/policy.py

```diff
@@ -1132,21 +1149,37 @@ def export_checkpoint(
                 old="Policy.export_checkpoint(filename_prefix=...)",
                 error=True,
             )
+        assert checkpoint_format in ["cloudpickle", "msgpack"], (
+            f"Value of `checkpoint_format` ({checkpoint_format}) must either be "
+            "'cloudpickle' or 'msgpack'!"
+        )
+
         if policy_state is None:
             policy_state = self.get_state()
-        policy_state["checkpoint_version"] = CHECKPOINT_VERSION
 
         # Write main policy state file.
         os.makedirs(export_dir, exist_ok=True)
-        with open(os.path.join(export_dir, "policy_state.pkl"), "w+b") as f:
-            pickle.dump(policy_state, f)
+        state_file = None
+        if checkpoint_format == "cloudpickle":
+            policy_state["checkpoint_version"] = CHECKPOINT_VERSION
+            state_file = "policy_state.pkl"
+            with open(os.path.join(export_dir, state_file), "w+b") as f:
+                pickle.dump(policy_state, f)
+        else:
```

Comment:
I think we can save ourselves an error here if we change this to:
```
elif checkpoint_format == "msgpack" else: raise error that checkpoint format is not reckognized. 
```

Reply:
Added this above:
```
        if checkpoint_format not in ["cloudpickle", "msgpack"]:
            raise ValueError(
                f"Value of `checkpoint_format` ({checkpoint_format}) must either be "
                "'cloudpickle' or 'msgpack'!"
            )
```

Reply:
perf